src/mongocxx/test/sdam-monitoring.cpp

```diff
@@ -241,13 +258,20 @@ TEST_CASE("SDAM Monitoring", "[sdam_monitoring]") {
         open_and_close_client(test_uri, apm_opts);
         REQUIRE(heartbeat_started_events > 0);
         REQUIRE(heartbeat_succeeded_events > 0);
+        REQUIRE(started_awaited_called);
+        REQUIRE(succeeded_awaited_called);
```

Comment:
This does not test that awaited is ever true. But this does test we're calling the underlying libmongoc function.